models.py

```diff
@@ -0,0 +1,12 @@
+import sqlalchemy
+
+META = sqlalchemy.MetaData()
+PEOPLE = sqlalchemy.Table(
```

Comment:
I'm not sure where have you got the code from but those are most definitely not constants.

https://docs.sqlalchemy.org/en/20/core/metadata.html